backend/src/main/java/team15/airbnb/reservation/application/ReservationService.java

```diff
@@ -61,6 +60,18 @@ public void reserve(Long accommodationId, Long userId, ReserveDto reserveDto) {
 		User user = userRepository.findById(userId);
 		Accommodation accommodation = accommodationRepository.findById(accommodationId);
 		int duration = calculateDuration(reserveDto.getCheckInDate(), reserveDto.getCheckOutDate());
+		/*
+		* 1. 예약하고자 하는 기간에 해당 숙소가 예약이 가능한지 여부 체크
+		* 2. 예약을 하다가 저장은 됐는데, 중간에 네트워크 이슈로 인하여, 클라이언트 측에 OK를 전달하지 못하였을 때
+		* 	- userId, accommodationId, checkInDate, checkOutDate 를 통해서 이미 DB에 저장이 되어있는 숙소인지 체크
+		* */
+		if (!validateReservation(accommodationId, reserveDto)) {
+			throw new IllegalStateException("해당 날짜는 다른 유저에게 예약이 되어 있습니다.");
+		}
```

Comment:
지금도 나쁘지는 않은데요. 이렇게 if 내부의 boolean 메서드라면 아래처럼 네이밍하는 것이 자연스럽게 읽혀서 좋을 것 같네요~
```suggestion
		if (!isAccommodationAvailable(accommodationId, reserveDto)) {
			throw new IllegalStateException("해당 날짜는 다른 유저에게 예약이 되어 있습니다.");
		}
```

Reply:
자연스럽게 읽힐 수 있는지 여부를 네이밍 기준으로 삼아보겠습니다!